### PR TITLE
Fix to OStatus delivery to be not so blocking to other tasks

### PR DESCRIPTION
**Pull request by annando**
While doing tests for my previous PR, I realized that the delivery process is massively blocked by not reachable OStatus contacts. We now are using the same date calculation method that we use with the queue. Additionally we are calling the push delivery from the queue with a lower priority.

This all should help increasing the delivery speed of messages.

### CODE REVIEW

**Comment by annando** [comment]
Now it works fine and should be faster than before.

File: src/Model/PushSubscriber.php [hunk 1]
```diff
@@ -24,7 +24,10 @@ public static function publishFeed($priority = PRIORITY_HIGH)
 			// We always handle retries with low priority
 			if ($subscriber["push"] > 1) {
 				$priority = PRIORITY_LOW;
+			} else {
+				$priority = $default_priority;
```
Comment (MrPetovan) [inline src/Model/PushSubscriber.php]: What was the issue?

Comment (annando) [reply src/Model/PushSubscriber.php]: It is partly related to issue https://github.com/friendica/friendica/issues/5043. While testing the fix I realised that it took really, really long until my test posts came through.

Comment (MrPetovan) [reply src/Model/PushSubscriber.php]: I was asking specifically about the add of the `$default_priority` variable in commit https://github.com/friendica/friendica/pull/5077/commits/435501449213958b41d3dbeeacafb4f534aef374

Comment (annando) [reply src/Model/PushSubscriber.php]: This answer does fit as well :-) "remote-self" contact related push subscriptions are handled with a lower priority - like the delivery.

Comment (MrPetovan) [reply src/Model/PushSubscriber.php]: No but you simply used `$priority` before, why did you need to add an intermediate variable on top of it? This is a code question, not a feature one.

Comment (annando) [reply src/Model/PushSubscriber.php]: It is in a loop. So we have to reset it with every loop, since it depends on the "push" level.

Comment (MrPetovan) [reply src/Model/PushSubscriber.php]: Ha, I see it now, thanks for the explanation!

Comment (annando) [reply src/Model/PushSubscriber.php]: BTW: I stumbled upon it as well.

**Pull request merged**